code/FsComposite/Composite.Simple/Data.fs

```diff
@@ -1,32 +1,37 @@
 ﻿namespace Composite.Simple
 
-open Composite.Core.Composite
-open Composite.Core.Processing
+open Composite.Common.DataTransformationHelper

```

Comment:
The Simple stuff belongs in the test project altogether. Consider moving.

Reply:
We have already done this in the "vi-config-file" branch, so merge with master will resolve this.